addons/mail/static/src/owl/components/chatter/chatter.scss

```diff
@@ -13,7 +13,7 @@
 .o_FormRenderer_chatterContainer {
     display: flex;
     flex: 1 1 auto;
-    width: 530px;
+    min-width: 530px;
```

Comment:
This rule should be split with a size breakpoint, it is not acceptable in mobile it should be responsive.